Directory.Build.props

```diff
@@ -12,7 +12,6 @@
   </PropertyGroup>
 
   <Import Condition="Exists('..\Directory.Build.props')" Project="..\Directory.Build.props" />
```

Comment:
what is this for?

Reply:
I think it's leftover from the old dir.props model, it used to be convention to always upload a higher-level dir.props if it existed. I'll delete it.